src/http/server.ts

```diff
@@ -851,15 +892,56 @@ export class Server {
    */
   protected async executeMiddlewareServerLevelAfterRequest(
     request: Drash.Http.Request,
-    response: Drash.Http.Response | null,
+    response: Drash.Http.Response | null = null,
   ): Promise<void> {
+    if (this.middleware.runtime) {
+      if (response) {
+        await this.executeMiddlewareServerLevelRuntimeAfterRequest(
+          request,
+          response,
+        );
+      }
+    }
+
     if (this.middleware.after_request != null) {
       for (const middleware of this.middleware.after_request) {
         await middleware(request, response!);
       }
     }
   }
 
+  /**
+   * Execute server level runtime middleware after requests. Runtime middleware
+   * requires compile time data.
+   *
+   * @param request - The request objecft.
+   * @param response - The response object.
+   *
+   * @returns A Drash response or a boolean. If it returns a boolean, it means
+   * it did or did not process properly (true being it processed; false being it
+   * did not process).
+   */
+  protected async executeMiddlewareServerLevelRuntimeAfterRequest(
+    request: Drash.Http.Request,
+    response: Drash.Http.Response,
+  ): Promise<void> {
```

Comment:
Doc block says it returns a drash response or boolean, but method return type is void

Reply:
93082a1 (it supposed to return void -- i removed the @return part)